### ISSUE TITLE
import/export and exchange rxn missing for dihydroorotate

### ISSUE THREAD

**Issue by johan-gson** [SysBioChalmers/Human-GEM#357]
### Description of the issue:
The metabolite (S)-dihydroorotate[c] is reported to be exported by cells (see https://www.nature.com/articles/s41467-018-08033-9 ), but there is no export rxn (and I assume there should be an import rxn as well). There is no exch rxn either. I have no idea about GPR though. This is happening in hypoxia.

#### Expected feature/value/output:
Just add the reactions I'm thinking, i.e.
(S)-dihydroorotate[c] <=> (S)-dihydroorotate[s]
(S)-dihydroorotate[s] <=> 

#### Reproducing these results:

**I hereby confirm that I have:**
- [ ] Tested my code on my own computer for running the model
- [ ] Done this analysis in the `main` branch of the repository
- [ ] Checked that a similar issue does not exist already

**Comment by JonathanRob** [comment]
Thanks for spotting this. Though I guess now we would abbreviate the compartment as `[e]` rather than `[s]`.

**Comment by mihai-sysbio** [comment]
Looking a bit more into this, the annotation for these reactions doesn't look straightforward.

A name search for `(S)-dihydroorotate` returned [CHEBI:30864](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:30864), but that is different from [CHEBI:17025](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:17025) which is the annotation of [MAM00180c](https://metabolicatlas.org/explore/Human-GEM/gem-browser/metabolite/MAM00180c). In fact, the CHEBI name for MAM00180c is `(S)-dihydroorotic acid`.

Overlooking the potentially misannotated metabolite, some [transport reactions for `(S)-dihydroorotate` seem to exist in MNX](https://beta.metanetx.org/cgi-bin/mnxweb/select_gpr?chem=MNXM252).

**Comment by johan-gson** [comment]
Yeah, it is with this for many compounds, that you can either talk about the ion or the acid (which includes the H+). It is the same with for example the amino acid glutamate/glutamic acid

**Comment by edkerk** [comment]
Thanks, this was a real gap. (S)-dihydroorotate ([MAM00180](https://metabolicatlas.org/explore/Human-GEM/gem-browser/metabolite/MAM00180c), [KEGG C00337](https://www.kegg.jp/entry/C00337)) existed only in the cytosol, so the model could not reproduce its secretion under hypoxia ([PMID:30643150](https://pubmed.ncbi.nlm.nih.gov/30643150/)). The extracellular form plus transport and exchange are now added, using `[e]` as suggested.

| New ID | Type | Reaction |
|---|---|---|
| MAM00180e | metabolite | (S)-dihydroorotate in the extracellular compartment |
| MAR20193 | transport | (S)-dihydroorotate[c] <=> (S)-dihydroorotate[e] |
| MAR20194 | exchange | (S)-dihydroorotate[e] <=> |

Both reactions are reversible, so import and export both work. The transport carries no GPR, since there is no established plasma-membrane transporter for dihydroorotate; if a specific carrier is identified it can be added later.

On the annotation point: MAM00180 carries [CHEBI:17025](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:17025) (the acid) rather than [CHEBI:30864](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:30864) (the anion). That is the general ion-versus-acid convention across the model rather than something specific to this metabolite, so it is left unchanged here instead of fixing one case in isolation.
